test/second_q/algorithms/excited_state_solvers/test_excited_states_solvers.py

```diff
@@ -139,6 +99,20 @@ def _solve_with_vqe_mes(self, converter: QubitConverter):
         results = esc.solve(self.electronic_structure_problem)
         self._assert_energies(results.computed_energies, self.reference_energies)
 
+    @data(
+        QubitConverter(JordanWignerMapper()),
+        QubitConverter(JordanWignerMapper(), z2symmetry_reduction="auto"),
+        QubitConverter(ParityMapper()),
+        QubitConverter(ParityMapper(), two_qubit_reduction=True),
+        QubitConverter(ParityMapper(), z2symmetry_reduction="auto"),
+        QubitConverter(ParityMapper(), two_qubit_reduction=True, z2symmetry_reduction="auto"),
+        QubitConverter(BravyiKitaevMapper()),
+        QubitConverter(BravyiKitaevMapper(), z2symmetry_reduction="auto"),
+    )
+    def test_vqe_mes(self, qubit_converter):
+        """Test VQEUCCSDFactory with QEOM + Various mappings"""
+        self._solve_with_vqe_mes(qubit_converter)
```

Comment:
I think you can just put the content of that method in here directly. Or is it still being used elsewhere and I am overlooking it?